modules/integrate-with-tenable.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-vulnerability-scanners.adoc
+:_module-type: PROCEDURE
+[id="integrate-with-tenable_{context}"]
+= Integrating with Tenable
+
+You can integrate {product-title} with Tenable for scanning images.
+
+.Prerequisites
+* You have Tenable *Access Key* and *Secret Key* for authentication.
```

Comment:
You could update this to "You have a Tenable **Access Key** and a **Secret Key**...".